src/types.d.ts

```diff
@@ -0,0 +1,4 @@
+interface Todo {
```

Comment:
`.d.ts` files are for external libraries. 

In most cases this should be `types.ts` and this interface should be exported

```ts
export interface Todo {
```

and import in other files
```ts
import { Todo } from './types'
```